AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -324,21 +324,21 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
 /**
- * @abstract Indicates that the node should fetch any external data, such as images.
+ * @abstract Indicates that the node should preload any external data, such as images.
  *
- * @discussion Subclasses may override this method to be notified when they should begin to fetch data. Fetching
+ * @discussion Subclasses may override this method to be notified when they should begin to preload. Fetching
  * should be done asynchronously. The node is also responsible for managing the memory of any data.
  * The data may be remote and accessed via the network, but could also be a local database query.
  */
-- (void)fetchData ASDISPLAYNODE_REQUIRES_SUPER;
+- (void)preload ASDISPLAYNODE_REQUIRES_SUPER;
 
 /**
- * Provides an opportunity to clear any fetched data (e.g. remote / network or database-queried) on the current node.
+ * Provides an opportunity to clear any preloaded data (e.g. remote / network or database-queried) on the current node.
  *
- * @discussion This will not clear data recursively for all subnodes. Either call -recursivelyClearFetchedData or
- * selectively clear fetched data.
+ * @discussion This will not clear data recursively for all subnodes. Either call -recursivelyClearPreloadedData or
+ * selectively clear preloaded data.
  */
-- (void)clearFetchedData ASDISPLAYNODE_REQUIRES_SUPER;
+- (void)clearPreloadedData ASDISPLAYNODE_REQUIRES_SUPER;
```

Comment:
Should we get rid of this method?  Is didExitPreloadState enough?  I think it probably is.